samcli/lib/generated_sample_events/event-mapping.json

```diff
@@ -190,7 +190,35 @@
           "default": "us-east-1.amazonaws.com"
         }
       }
-    }
+    },
+    "http-api-proxy": {
+      "filename": "HttpApi",
+      "help": "Generates an Amazon API Gateway Http API Event",
+      "tags": {
+        "body": {
+          "type": "string",
+          "default": "{\"test\":\"body\"}",
+          "encoding": "base64"
+        },
+        "stage": {
+          "type": "string",
+          "default": "$default"
+        },
+        "method": {
+          "type": "string",
+          "default": "POST"
+        },
+        "path": {
+          "type": "string",
+          "default": "path/to/resource"
+        },
+        "account-id": {
+          "default": "123456789012"
+        },
+        "dns-suffix": {
```

Comment:
this parameter is not used.

Reply:
dns-suffix is used as `dns_suffix` due to how the templating works.:

```json
    "domainName": "id.execute-api.{{{dns_suffix}}}",
```

Reply:
sorry it is my bad